packages/pkg-unpacker.vm/tools/chocolateyinstall.ps1

```diff
@@ -6,7 +6,7 @@ try {
     $category = 'Packers'
     $zipUrl = 'https://github.com/LockBlock-dev/pkg-unpacker/archive/b1fd5200e1bf656dedef6817c177c8bb2dc38028.zip'
     $zipSha256 = '6eed1d492d37ca3934a3bc838c2256719a3e78ccf72ce1b1ca07684519ace16c'
-    $powershellCommand = "Write-Output '> node unpack.js'; node unpack.js"
+    $powershellCommand = "npm install; get-content README.md"
```

Comment:
Removing the `Write-Output` is due to having the command be displayed when it is executed once https://github.com/mandiant/VM-Packages/pull/871 is merged

I can remove the `get-content README.md` piece, if you'd like. I added that so a user would have some sort of help shown to them on how to use the tool, because otherwise there is no way for a user to know how to use it since there is no help flag.

I can try to find way to make it such that it only prints the `Usage piece`: https://github.com/LockBlock-dev/pkg-unpacker/?tab=readme-ov-file#usage, or I can just leave it as where the user will have to figure out how to use it themselves. 😬

Reply:
I like to use (and specially to display) `node unpack.js` so that the user knows what command to run. So after #871, `npm install; node unpack.js` would work. @mandiant/vms what do you think?

Reply:
Oooh, that makes sense. So instead of displaying the help, you want it to display the potential command to use.

That could work, though it still doesn't give the user a full help/usage or anything to tell them what they actually need to use, which is an unfortunate thing about the tool. 😕